README.md

```diff
@@ -0,0 +1,180 @@
+# aws-cli
+
+[![Build Status](https://travis-ci.org/aws/aws-cli.svg?branch=develop)](https://travis-ci.org/aws/aws-cli)
+
+[![Gitter](https://badges.gitter.im/aws/aws-cli.svg)](https://gitter.im/aws/aws-cli)
+
+This package provides a unified command line interface to Amazon Web Services.
+
+Jump to:
+
+- [Getting Started](#getting-started)
+- [Getting Help](#getting-help)
+- [More Resources](#more-resources)
+
+## Getting Started
+
+This README is for the AWS CLI version 1. If you are looking for information about the AWS CLI version 2, please visit the [v2 branch](https://https://github.com/aws/aws-cli/tree/v2).
```

Comment:
I think the link for v2 here has two `https://` in it. It might be worth running a link checker over these to make sure we did not miss any more links.